plat/kvm/Config.uk

```diff
@@ -109,8 +112,10 @@ config KVM_VMM_FIRECRACKER
 	select HAVE_MMIO
 	select HAVE_RANDOM
 	select VIRTIO_MMIO_LINUX_COMPAT_CMDLINE
-	imply LIBUKCONSOLE_NS16550 if ARCH_ARM_64
+	imply LIBUKCONSOLE_NS16550 if (ARCH_ARM_64 || ARCH_X86_64)
```

Comment:
Due to the `depends on` of `KVM_VMM_FIRECRACKER`, `ARCH_ARM_64` and `ARCH_X86_64` should be already covered here.